### ISSUE TITLE
21/22 Player list compare vs last 5 years

### ISSUE THREAD

**Issue by rtibbsy** [vaastav/Fantasy-Premier-League#96]
Hello - is it possible to get a list of all players in the game as of today and then a breakdown of each players output from each year going back 5 seasons?

**Comment by vaastav** [comment]
Yes, I think its possible to produce such a list. I'll put it on my todo list.
